README.md

```diff
@@ -23,8 +23,8 @@ with all required dependencies and then run the script to preprocess the project
 
 # Publish
 
-Simply commit your changes and do a git push.  There is also `make publish` which integrates with the rook `Makefile`
-nicely, and is meant to be run in conjunction with Jenkins CI -- and expects environment variable `DOCS_VERSION`.
+Simply commit your changes and do a git push, the CI will do the reset that is need to get the changes build and pushed to the `gh-pages` branch.
+To manually publish changes, this assumes you have full write access to the repository, run  `make publish`.
```

Comment:
We don't need to document the manual publish, right? If someone wants to see the implementation they can look at the CI workflow.
```suggestion
```